src/gui/modals/SetupDialog.cpp

```diff
@@ -267,7 +267,12 @@ SetupDialog::SetupDialog(ConfigTabs tab_to_open) :
 		// Get locale extracted by filename.
 		fileNames[i].truncate(fileNames[i].lastIndexOf('.'));
 		m_languages.append(fileNames[i]);
-		QString lang = QLocale(m_languages.last()).nativeLanguageName();
+		QLocale locale(m_languages.last());
+		QString lang = locale.nativeLanguageName();
+		// If the language code is invalid, display the filename
+		if (locale.language() == QLocale::C) { lang = fileNames[i]; }
```

Comment:
So you display the `qt_` files as their filename? I understood this like they can not be used as a language,so why not hide them completely?

Reply:
> So you display the `qt_` files as their filename? I understood this like they can not be used as a language,so why not hide them completely?

I think so.

Reply:
Good point, done

Reply:
I think the fix is invalid, because now, `m_languages` and `changeLang` will have different sizes and the indices don't match. The mismatching indices would bring errors, e.g. in 

```c++
	connect(changeLang, SIGNAL(currentIndexChanged(int)),
			this, SLOT(setLanguage(int)));
```

Reply:
Thanks for catching that! Fixed.